templates/web/smidsy/about/faq.html

```diff
@@ -3,95 +3,174 @@
+
+    <dt>My report is old and outdated. Can I now delete it?</dt>
+    <dd>We have a policy of retaining all Collideoscope reports because they help create an evidence base of cycling incidents across the country. This data is useful for town planners and researchers who can use it as hard evidence to back up decisions and recommendations for safer road policies.</dd>
+    <dd>As new reports are made, older ones drop off the first page for any given area, and are less likely to be discovered by those visiting the site.</dd>
+
+    <dt>Can you delete my name from the site?</dt>
+    <dd>Yes. Please <a href="/about/privacy">check our Privacy Policy</a> for full details of your rights under the GDPR and information about how to request this.</dd>
+</dl>
+
+<h2 name="accounts">Accounts</h2>
+
+<dl>
+    <dt>Do I need to register with Collideoscope to use it?</dt>
+    <dd>No – you can send a report without registering. When you submit your report, we’ll send you an email with a confirmation link.</dd>
+    <dd>If you use Collideoscope regularly, you can avoid this step by creating an account. Once you’ve signed in, your reports will be sent as soon as you click ‘submit’.</dd>
+    <dd>You’ll also be able to see all previous reports you’ve made on one page.</dd>
+
+    <dt>How do I register?</dt>
+    <dd>Visit <a href="https://www.collideoscope.org.uk/auth">www.collideoscope.org.uk/auth</a> and choose ‘No, let me sign in by email’. You can set a password at that point.</dd>
+
+    <dt>I’ve forgotten my password</dt>
+    <dd>Don’t worry! Visit <a href="https://www.collideoscope.org.uk/auth">www.collideoscope.org.uk/auth</a> and choose ‘No, let me sign in by email’ – then put a new password in the box. You’ll receive an email to confirm the change.</dd>
+
+    <dt>I want to change my password or email address</dt>
+    <dd>You can do this from your <a href="/my">account page</a>.</dd>
+</dl>
+
+<h2 name="councils">Councils</h2>
+
+<dl>
+    <dt>I work for a council. How can we access data from Collideoscope?</dt>
+    <dd>If your council is within Great Britain, we are already sending reports to you as soon as they are submitted by our users. Please <a href="/contact">drop us a line</a> if you’d like to check or change which email address we are using, or if you’d like an address added.</dd>
+
+    <dt>Can we get more or different data in reports?</dt>
+    <dd><a href="/contact">Contact us</a> to let us know what you need; it’s possible we’ll be able to add to or modify the report form for your area.</dd>
+
+    <dt>Can we modify the text you send to people who make reports?</dt>
+    <dd>Yes! If there’s something particular you’d like to include about, for example, your commitment to road safety, or plans for future improvements, just drop us a line and we can change our emails accordingly.</dd>
```

Comment:
This facility doesn't currently exist (I don't know if we want to leave this text here to see if anyone asks for it).

Reply:
Yeah, I figured we’d leave this, just to see whether someone wants it. Given sufficient time/budget, we could implement it!